server/irmaserver/helpers.go

```diff
@@ -343,17 +344,59 @@ func (session *session) getRequest() (irma.SessionRequest, error) {
 	return cpy.(*irma.IssuanceRequest), nil
 }
 
+func (s *sessionData) hash() [32]byte {
+	return sha256.Sum256([]byte(fmt.Sprintf("%v", s)))
```

Comment:
If I read it correctly the panic was swallowed by fmt's `print.go`:
```
			case Stringer:
				handled = true
				defer p.catchPanic(p.arg, verb, "String")
				p.fmtString(v.String(), verb)
				return
			}
```

So why would we only see this in the debugger?
Possibly this is just a debugger bug which is stuck with the nil derefence. See [https://github.com/go-delve/delve/issues/852](https://github.com/go-delve/delve/issues/852)

fixed now by using JSON marshalling instead of `fmt.Sprintf`